### PR TITLE
feat: Add documentation about object storage endpoint

### PR DESCRIPTION
**Pull request by Geun-Oh**
feat: Add mapRegionWithEnv util func

docs: Add env configuration guide

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -74,10 +74,14 @@ provider "ncloud" {
 
 ### Environment variables
 
-You can provide your credentials via `NCLOUD_ACCESS_KEY` and `NCLOUD_SECRET_KEY`,
-environment variables, representing your Ncloud Access Key and Secret Key, respectively.
-`NCLOUD_REGION` is also used, if applicable:
+* `NCLOUD_ACCESS_KEY` - (Required) Ncloud access key.
```
Comment (youngmn) [inline docs/index.md]: (Optional, Required if `access_key` is not provided)

File: docs/index.md [hunk 2]
```diff
@@ -74,10 +74,14 @@ provider "ncloud" {
...
+* `NCLOUD_SECRET_KEY` - (Required) Ncloud secret key.
```
Comment (youngmn) [inline docs/index.md]: (Optional, Required if `secret_key` is not provided)

File: docs/index.md [hunk 3]
```diff
@@ -74,10 +74,14 @@ provider "ncloud" {
...
+* `NCLOUD_REGION` - (Required) Ncloud region.
```
Comment (youngmn) [inline docs/index.md]: (Optional, Required if `region` is not provided)

**Review by wonchulee — APPROVED** [review]
👍

**Review by youngmn — APPROVED** [review]

**Pull request merged**